README.md

```diff
@@ -279,6 +279,8 @@ left out of this section.
   * **NOT SUPPORTED.** See [#101](https://github.com/citp/OpenWPM/issues/101).
   * Set to `True` to enable Firefox's built-in
     [Tracking Protection](https://developer.mozilla.org/en-US/Firefox/Privacy/Tracking_Protection).
+* `no-popups`
```

Comment:
Is my understanding correct that these two prefs would completely prevent a site from being able to create new tabs/windows? If so, let's call this `force-single-tab` and update the description to "Set to `True` to prevent pages from opening new windows/tabs, such as those that would result from a call to `window.open`."

Reply:
At least I hope that these two lines cause this behaviour, but trying to circumvent this restriction seems to be a famous research topic on the dark side of the web.